### DISCUSSION TITLE
Phone name display during session

### DISCUSSION THREAD

**Discussion by Vasilis553** [rustdesk/rustdesk#8643] [General]
### Bug Description

Hello ex.connect with my xiaomi phone in a Win11 computer it's not shows the name of device in CM example billyxiaomi  but the name of the phones model like redmi-19x... Is any way to add an optine for costum nickname during sessions? 

### How to Reproduce

-

### Expected Behavior

-

### Operating system(s) on local side and remote side

Android==win10,11

### RustDesk Version(s) on local side and remote side

1.2.7

### Screenshots

-

### Additional Context

-

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/3507
